cypress-tests/cypress/utils/State.js

```diff
@@ -5,6 +5,12 @@ class State {
     this.data["connectorId"] = Cypress.env("CONNECTOR");
     this.data["baseUrl"] = Cypress.env("BASEURL");
     this.data["adminApiKey"] = Cypress.env("ADMINAPIKEY");
+    this.data["profileId"] = Cypress.env("PROFILE_ID");
+    this.data["stripeMcaId"] = Cypress.env("STRIPE_MCA_ID");
+    this.data["adyenMcaId"] = Cypress.env("ADYEN_MCA_ID");
+    this.data["routingApiKey"] = Cypress.env("ROUTING_API_KEY");
+    this.data["email"] = Cypress.env("HS_EMAIL");
+    this.data["password"] = Cypress.env("HS_PASSWORD");
```

Comment:
I would expect this to be fetched from `connector_creds` as it is not good to have multiple creds read from `env`. As of now, we can have it in the same creds file itself.

Also, if you do not pass values through `env` for routing, the values for these `env`s will be `undefined`.

Reply:
having them put in creds_json would require to handle these data separately from the credentials and that too for each connector creation irrelevant of whether we need it or not. 

If I were to even put these data in a json file, it would be better to have them in a seperate json just to have a clear distinction on their purpose.

In terms of handling the envs, these will be obselete in the next refactor since the idea is to get the merchant_id, profile_id, mca_id and api_key through api calls to the server.